tools/hess/hess_astro_tool.xml

```diff
@@ -0,0 +1,217 @@
+        <param name="src_name" value="Crab" />
+        <param name="RA" value="83.6287" />
+        <param name="DEC" value="22.0147" />
+        <param name="T1" value="2000-10-09T13:16:00.0" />
+        <param name="T2" value="2022-10-10T13:16:00.0" />
+        <param name="Radius" value="2.5" />
+        <param name="pixsize" value="0.1" />
+        <param name="Emin" value="100.0" />
+        <param name="Emax" value="10000.0" />
+      </conditional>
+      <assert_stdout>
+        <has_text text="*** Job finished successfully ***" />
+      </assert_stdout>
+    </test>
+    <test expect_num_outputs="2">
+      <conditional name="_data_product">
+        <param name="_selector" value="Spectrum" />
+        <param name="src_name" value="PKS 2155-304" />
+        <param name="RA" value="329.716938" />
+        <param name="DEC" value="-30.225588" />
+        <param name="T1" value="2000-10-09T13:16:00.0" />
+        <param name="T2" value="2022-10-10T13:16:00.0" />
+        <param name="Radius" value="2.5" />
+        <param name="R_s" value="0.2" />
+        <param name="Emin" value="100.0" />
+        <param name="Emax" value="10000.0" />
+        <param name="NEbins" value="20" />
+      </conditional>
+      <assert_stdout>
+        <has_text text="*** Job finished successfully ***" />
+      </assert_stdout>
+    </test>
+    <test expect_num_outputs="2">
+      <conditional name="_data_product">
+        <param name="_selector" value="Spectrum_gammapy" />
+        <param name="src_name" value="PKS 2155-304" />
+        <param name="RA" value="329.716938" />
+        <param name="DEC" value="-30.225588" />
+        <param name="T1" value="2000-10-09T13:16:00.0" />
+        <param name="T2" value="2022-10-10T13:16:00.0" />
+        <param name="Radius" value="2.5" />
+        <param name="R_s" value="0.5" />
+        <param name="Emin" value="100.0" />
+        <param name="Emax" value="10000.0" />
+        <param name="NEbins" value="20" />
+      </conditional>
+      <assert_stdout>
+        <has_text text="*** Job finished successfully ***" />
+      </assert_stdout>
+    </test>
+    <test expect_num_outputs="2">
+      <conditional name="_data_product">
+        <param name="_selector" value="Lightcurve" />
+        <param name="src_name" value="Crab" />
+        <param name="RA" value="83.6287" />
+        <param name="DEC" value="22.0147" />
+        <param name="T1" value="2003-10-09T13:16:00.0" />
+        <param name="T2" value="2005-10-10T13:16:00.0" />
+        <param name="Radius" value="2.5" />
+        <param name="R_s" value="0.2" />
+        <param name="Emin" value="100.0" />
+        <param name="Emax" value="10000.0" />
+        <param name="NTbins" value="10" />
+      </conditional>
+      <assert_stdout>
+        <has_text text="*** Job finished successfully ***" />
+      </assert_stdout>
+    </test>
+  </tests>
+  <help>This service provides analysis of publicly available sample &#8220;Data Level
```

Comment:
If it makes your life easier you could use CDATA here. This would make reading the source easier.

Reply:
You mean, to avoid e.g. symbol codes? Agree
Need to figure out when it gets converted to codes. This help text was originally a markdown file.

Reply:
Its a long-standing issues that we should support `<help type="markdown">`. This way you could just copy it over.
We will get there at some point.